src/main/java/com/enderio/EnderIO.java

```diff
@@ -100,6 +101,13 @@ public EnderIO() {
         modEventBus.addListener(SoulVialItem::onCommonSetup);
         IntegrationManager.addIntegration(EnderIOSelfIntegration.INSTANCE);
         new UseGliderTrigger().register();
+        new PaintingTrigger().register();
+
+        // Decor
+        DecorBlocks.register();
```

Comment:
I'm in two minds here:

Either we should merge all of these down into a single init class again
-or-
we could further separate these out into more divided chunks to make them easier to navigate.

I'm happy for whichever as long as we're consistent :)

Reply:
oh no, work